src/platform/forms-system/src/js/widgets/RadioWidget.jsx

```diff
@@ -18,6 +20,13 @@ export default function RadioWidget({
     content = <NestedContent />;
   }
 
+  if (focusedElement) {
+    // focus is intermittently lost
```

Comment:
Do we know why this is happening? Feels like this might be treating the symptom rather than the core issue that conditionally displaying information is hijacking focus and not announcing the assistive technology.

Reply:
@timwright12 On the note of treating the symptom as opposed to the core issue-- I'll be talking to the team about the design itself to see if we can explore separating the expanded items into a separate page. This may be better approached as a design problem instead of an accessibility remediation challenge.

Reply:
I haven't tested other forms, but I believe this is an inherent problem with the radio widget. When a radio is selected, it wraps the button in an `<ExpandingGroup>` element, which replaces the existing `<input>` with a newly rendered `<input>` and focus becomes lost

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src%2Fplatform%2Fforms-system%2Fsrc%2Fjs%2Fwidgets%2FRadioWidget.jsx#L42-L51

Reply:
If that's the case would it be better to not use `<ExpandingGroup>` and rather rely on ARIA states to report the content?
```
{radioButton}
<div aria-labelledby={`${id}_${i}`} aria-live="polite" aria-atomic="true" className="schemaform-radio-indent" open={checked} key={option.value}>{content}</div>
``

Reply:
@timwright12 @Mottie Just talked to @luke-gcio , we got tentative approval to split this into multiple pages instead of having to deal with expanding/collapsing new inputs, multiple fieldsets, etc 👍

Reply:
@Mottie I can't seem to replicate the behavior in the screenshot above using keyboard on Chrome where you're able to navigate between no -> yes -> yes without hiccups.

https://user-images.githubusercontent.com/14154792/103542799-841ea480-4e6b-11eb-9181-7f083286cf5b.mov

I also tested with Firefox and the same issue emerges on the second time going through the options. On the first time using the keyboard, it works as expected, but upon the field group disappearing once and reappearing again it reemerges. Any ideas on what might be happening? 😕

Reply:
I just tested the code in this PR in Chrome and the focus does appear to stay within the radio group. In master, I see the same behavior as in the screen recording.

Reply:
@Mottie Sorry, my brain isn't working this morning, I was testing in staging. If the focus works in the PR we should be good to go!

Reply:
> If that's the case would it be better to not use `<ExpandingGroup>` and rather rely on ARIA states to report the content?

@timwright12 Yes and no, I  think.

My understanding of this approach is that all possible expanded content exists in the DOM, but is `display: hidden` (or some such) and uses ARIA roles to "show / hide" it.

I think the problem we'd run into there is that some expanded content may be fields with validation. When we try to move to the next page, it'll run the validation on the whole page—including those hidden fields—and we won't be able to continue, but also won't be able to see the validation error because the "offending" field is hidden. This turns that solution into a bigger thing where we have to figure out which validations are the ones that we need to run, etc. etc.

All that said, if the `ExpandingGroup` is the culprit for focus loss, it feels like the solution should be in there, not in the component which uses it (`RadioWidget`). Is that a feasible approach, @Mottie?

Reply:
I'm not sure what the issue was, I should have tested the radio group with different content. Either way we had to abandon this method because the radio group wrapping `fieldset` would include the new form elements and make it more difficult for a screenreader to differentiate the groups.

Reply:
Ahhh, interesting. Does this PR need to be closed, then, or are you working on a different solution in this branch?

Reply:
We could close this PR. It isn't applicable to HLR any more, but it may be an issue with other forms?